### PR TITLE
More Bug Fixes

### LINKED ISSUE #5427: Custom sprite mislabel
There is a single remaining mislabel in the custom synthetics file which causes the eyes for a specific module to not show up, as seen here -

It should be labeled, like the other ones, "eyes-kyres1-Combat"

### LINKED ISSUE #5463: Cacoa is missing from the MegaSeed Servitor, packets are bugged to be nameless, spriteless, and have no contents

### LINKED ISSUE #5466: Nanomed in Xenobiology
There seems to be a small issue with the Nanomed in Xenobiology. As a xenobiologist I can't get any meds out of it.

### PR DESCRIPTION
**Pull request by dragoslava**
- rscadd: Fixes #5466
  - rscadd: Fixes #5427
  - rscadd: Fixes #5463

### CODE REVIEW

**Comment by BotBOREALIS** [comment]
Icons with diff:
<details><summary>icons/mob/custom_synthetic.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
kyres1-Combat-eyes|kyres1-Combat-eyes||Removed
kyres1-Combat-eyes[1]|![kyres1-Combat-eyes[1]](http://images.aurorastation.org/images/388efe1ddfd64d744253908bb587f7a8.png)||Removed
kyres1-Combat-eyes[2]|![kyres1-Combat-eyes[2]](http://images.aurorastation.org/images/eac0ae8c9c3ea9fca2fa1a4d95e0d7dc.png)||Removed
kyres1-Combat-eyes[3]|![kyres1-Combat-eyes[3]](http://images.aurorastation.org/images/44dbee75019481bcdf9cf0333d0acc14.png)||Removed
eyes-kyres1-Combat||eyes-kyres1-Combat|Created
eyes-kyres1-Combat[1]||![eyes-kyres1-Combat[1]](http://images.aurorastation.org/images/388efe1ddfd64d744253908bb587f7a8.png)|Created
eyes-kyres1-Combat[2]||![eyes-kyres1-Combat[2]](http://images.aurorastation.org/images/eac0ae8c9c3ea9fca2fa1a4d95e0d7dc.png)|Created
eyes-kyres1-Combat[3]||![eyes-kyres1-Combat[3]](http://images.aurorastation.org/images/44dbee75019481bcdf9cf0333d0acc14.png)|Created
</details>
<details><summary>icons/obj/seeds.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
cacao||cacao|Created
</details>

**Comment by dragoslava** [comment]
!review

File: html/changelogs/dragochangelog.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Fixes Xenobio Nanomed Access."
+  - rscadd: "Fixes Kyres Sprite name."
+  - rscadd: "Fixes Cacoa Beans, gave them a sprite and proper name."
```
Comment (HeyBanditoz) [inline html/changelogs/dragochangelog.yml]: Use `bugfix` instead. Is a changelog even required for this?

File: maps/aurora/aurora-3_sublevel.dmm [hunk 2]
```diff
@@ -21140,7 +21140,9 @@
 	},
 /obj/effect/floor_decal/industrial/outline/blue,
 /obj/machinery/vending/wallmed1{
-	pixel_y = 32
+	pixel_y = 32;
+	req_access = null;
```
Comment (HeyBanditoz) [inline maps/aurora/aurora-3_sublevel.dmm]: Should be safe removing this line. By default `req_access` is just `list()`.

File: html/changelogs/dragochangelog.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Fixes Xenobio Nanomed Access."
+  - rscadd: "Fixes Kyres Sprite name."
```
Comment (Arrow768) [inline html/changelogs/dragochangelog.yml]: That is not relevant for the larger playerbase -> Doesnt get a changelog.

File: html/changelogs/dragochangelog.yml (hunk 1 above)
Comment (Sindorman) [reply html/changelogs/dragochangelog.yml]: If changes are visible it needs changelog. Sometimea even background fixes need change log

File: html/changelogs/dragochangelog.yml (hunk 3 above)
Comment (Sindorman) [reply html/changelogs/dragochangelog.yml]: Agree with Arrow

**Comment by Menown** [comment]
Was there a reason behind the renaming? Whomever renamed it to cacao was the one that ruined the whole seed string, and everything. Cocoa already had its own sprites.

File: html/changelogs/dragochangelog.yml (hunk 3 above)
Comment (Sindorman) [reply html/changelogs/dragochangelog.yml]:
```suggestion
```

File: maps/aurora/aurora-3_sublevel.dmm (hunk 2 above)
Comment (Sindorman) [reply maps/aurora/aurora-3_sublevel.dmm]:
```suggestion
```

File: html/changelogs/dragochangelog.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Fixes Xenobio Nanomed Access."
```
Comment (Sindorman) [inline html/changelogs/dragochangelog.yml]:
```suggestion
  - bugfix: "Fixes Xenobio Nanomed Access."
```

File: html/changelogs/dragochangelog.yml (hunk 1 above)
Comment (Sindorman) [inline html/changelogs/dragochangelog.yml]:
```suggestion
  - bugfix: "Fixes Cacoa Beans, gave them a sprite and proper name."
```

**Comment by Arrow768** [comment]
It seems mapmerge pruned a few keys

**Comment by Alberyk** [comment]
Looks like all bugs here were already handled, closing then.

**Pull request closed without merging**